messaging/src/main/java/org/axonframework/common/property/AbstractMethodPropertyAccessStrategy.java

```diff
@@ -21,6 +21,8 @@
 
```

Comment:
Nit: I am still missing the copyright adjustment in this file to increase the year from 2018 to 2024.

Reply:
sorry for that mistake. I changed it now